diff --git a/src/modules/cash/CashModuleCore.sol b/src/modules/cash/CashModuleCore.sol
index 09c16b9..1daaef4 100644
--- a/src/modules/cash/CashModuleCore.sol
+++ b/src/modules/cash/CashModuleCore.sol
@@ -4,7 +4,7 @@ pragma solidity ^0.8.28;
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
 import { MessageHashUtils } from "@openzeppelin/contracts/utils/cryptography/MessageHashUtils.sol";
-import { EnumerableSetLib } from "solady/utils/EnumerableSetLib.sol";
+import { EnumerableSet as EnumerableSetLib } from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
 
 import { ICashEventEmitter } from "../../interfaces/ICashEventEmitter.sol";
 import { Mode, BinSponsor, SafeCashConfig, SafeData, SafeTiers, WithdrawalRequest, TokenDataInUsd, Cashback, CashbackTokens } from "../../interfaces/ICashModule.sol";
