diff --git a/src/debt-manager/DebtManagerStorageContract.sol b/src/debt-manager/DebtManagerStorageContract.sol
index f4cae31..08f5b9b 100644
--- a/src/debt-manager/DebtManagerStorageContract.sol
+++ b/src/debt-manager/DebtManagerStorageContract.sol
@@ -3,7 +3,7 @@ pragma solidity ^0.8.28;
 
 import { IERC20Metadata } from "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
 import { IERC20, SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
-import { EnumerableSetLib } from "solady/utils/EnumerableSetLib.sol";
+import { EnumerableSet as EnumerableSetLib } from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
 import { Math } from "@openzeppelin/contracts/utils/math/Math.sol";
 
 import { IDebtManager } from "../interfaces/IDebtManager.sol";
