diff --git a/src/safe/EtherFiSafeFactory.sol b/src/safe/EtherFiSafeFactory.sol
index 15acdd2..e84c543 100644
--- a/src/safe/EtherFiSafeFactory.sol
+++ b/src/safe/EtherFiSafeFactory.sol
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.28;
 
 import { BeaconFactory, UpgradeableBeacon } from "../beacon-factory/BeaconFactory.sol";
-import { EnumerableSetLib } from "solady/utils/EnumerableSetLib.sol";
+import { EnumerableSet as EnumerableSetLib } from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
 
 import { EtherFiSafe } from "./EtherFiSafe.sol";

