diff --git a/src/top-up/TopUpFactory.sol b/src/top-up/TopUpFactory.sol
index cd7026b..9413c79 100644
--- a/src/top-up/TopUpFactory.sol
+++ b/src/top-up/TopUpFactory.sol
@@ -3,7 +3,7 @@ pragma solidity ^0.8.28;
 
 import { IERC4626 } from "@openzeppelin/contracts/interfaces/IERC4626.sol";
 import { IERC20, SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
-import { EnumerableSetLib } from "solady/utils/EnumerableSetLib.sol";
+import { EnumerableSet as EnumerableSetLib } from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
 
 import { BeaconFactory, UpgradeableBeacon } from "../beacon-factory/BeaconFactory.sol";
 import { ITopUpFactory } from "../interfaces/ITopUpFactory.sol";
